doc/man/man8/daos.8

```diff
@@ -122,6 +126,21 @@ The \fBRESOURCE\fRs, respective \fBCOMMAND\fRs and \fBOPTION\fRs supported by \f
 .br
 				      K (KB), M (MB), G (GB), T (TB), P (PB), E (EB)
 .TP
+.I container \fBOPTION\fRs (create):
+	  \fB--properties=\fR<name>:<value>[,<name>:<value>,...]      (optional) container properties
+.br
+				      Supported properties names:
+.br
+				      \fBlabel\fR (can be any string)
+.br
+				      \fBcksum\fR checksum type (can be {off,crc{16,32,64}})
```

Comment:
cksum: valid input also includes "sha1" I think (looking at the code)

Reply:
Nope, still not. See current/master source code using DAOS_PROP_CO_CSUM_SHA1 ... But ok, let's add it already if you want, if it is likely to be supported soon?

Reply:
I see yes you are right it is not supported yet. But it is in the help output, so perhaps adding it to the manpage now is good to be consistent with that. As I recall early on in the tools proposal we decided the help output should be as complete as possible (given what was understood initially) with placeholders for that known future functionality.